crates/proof-of-sql/src/base/database/column.rs

```diff
@@ -1201,4 +1214,46 @@ mod tests {
         let round_trip_owned: OwnedColumn<TestScalar> = (&column).into();
         assert_eq!(owned_varbinary, round_trip_owned);
     }
+
+    #[test]
+    fn we_can_get_min_scalar() {
+        assert_eq!(
+            TryInto::<i8>::try_into(ColumnType::TinyInt.min_scalar::<TestScalar>().unwrap())
+                .unwrap(),
+            i8::MIN
+        );
```

Comment:
This is more direct and compact:
```suggestion
        assert_eq!(ColumnType::TinyInt.min_scalar(), Some(TestScalar::from(i8::MIN)));
```